lib/qa/authorities/discogs.rb

```diff
@@ -0,0 +1,130 @@
+require 'rdf'
+module Qa::Authorities
+  class Discogs < Base
+    extend ActiveSupport::Autoload
+    autoload :DiscogsTranslation
+    autoload :DiscogsUtils
+    autoload :DiscogsWorksBuilder
+    autoload :DiscogsInstanceBuilder
+    include WebServiceBase
+    include DiscogsTranslation
+
+    class_attribute :discogs_secret, :discogs_key
+    attr_accessor :primary_artists
+
+    def initialize
+      self.primary_artists = []
+    end
+
+    def search(q, tc)
```

Comment:
Can you add yardoc for parameters?  I know existing code is not fully documented, but the standard going forward is to include yardoc for new code.

https://www.rubydoc.info/gems/yard/file/docs/Tags.md#List_of_Available_Tags

This file is well documented and can serve as an example.

Reply:
Did you look at using AuthorityWithSubAuthority?

Reply:
done